src/fragments/lib/push-notifications/ios/getting_started/20_set_entitlements.mdx

```diff
@@ -0,0 +1,42 @@
+## Set Entitlements
+
+Using Amazon Pinpoint with APNs requires the following capabilities
+- Push Notifications
+- Background Processing -> Remote Notifications (**iOS only**)
```

Comment:
Should we display these differences in a` <BlockSwitcher> `instead,  as we do with code?

Reply:
I like this.  Done.